packages/ticketing/src/pdf-tickets/defaultOrderPDFRenderer.ts

```diff
@@ -0,0 +1,105 @@
+import { TicketingAPI } from '../types.js';
```

Comment:
we should call this file defaultTicketReceiptRenderer and change the params/fn names accordingly now we also support returning other media types.